### ISSUE TITLE
How to specify Firefox version in Selenium 3.4.0 Grid

### ISSUE THREAD

**Issue by dordnung** [SeleniumHQ/selenium#3941]
## Meta -
OS:  
Windows

Selenium Version:  
3.4.0

Browser:  
Firefox

Browser Version:  
52

## Expected Behavior -
The Selenium remote webdriver can choose a node with the correct Firefox version by setting the version in the capabilities

## Actual Behavior -
Server responses with: "Unrecognized channel: 52"
The problem comes with these two commits: 
https://github.com/SeleniumHQ/selenium/commit/d773d2ae4ae2c17b8983bfdb25994490ef35b805#diff-1945a3d6d32eb94f85054cf3218bb7f1
https://github.com/SeleniumHQ/selenium/commit/d2378dd1d1b15c0e87ddc293dd7e74ef53443dd2#diff-230a6397be96d5d56ea1fd43ffda7b0d

Now the version in the capabilities has to be a channel and not a version anymore. Was this intended? If so, how can i specify a Firefox version from the Selenium Grid?

## Steps to reproduce -
```
DesiredCapabilities capabilities = DesiredCapabilities.firefox();
capabilities.setPlatform(Platform.LINUX);
capabilities.setVersion("52");
new RemoteWebDriver(this.seleniumHubUrl, capabilities);
```

**Comment by schmidtkp** [comment]
FWIW: If you **_do not_** call `capabilities.setVersion(version)` to avoid the "Unrecognized channel: version" error, but **_still_** call `capabilities.setCapability(CapabilityType.BROWSER_VERSION, version)` you get the following exception:
`org.openqa.selenium.SessionNotCreatedException: TypeError: Given browserVersion [object String] "", but current version is [object String] "53.0"`
You have to make sure both are not called to avoid an error.

**Comment by dordnung** [comment]
This seems to work, thank. But still weird, as there is still the DesiredCapabilities#setVersion method, but not setBrowserVersion method

**Comment by clubfest** [comment]
How does this translate for just a non-Java request? I tried adding the browserVersion field to the desired capabilities in JSON format, but it still gives the same error about unrecognized channel:

    {'version': '52',  'browserVersion': '52', 'marionette': True, 'browserName': 'firefox', 'platform': 'VISTA'}

**Comment by barancev** [comment]
Should be fixed in 3.5, please reopen the issue if it's not true.
